src/backend/functions/risk-edit.ts

```diff
@@ -23,38 +23,25 @@ const prisma = new PrismaClient();
 export const editRisk: Handler<FromSchema<typeof inputSchema>> = async ({ body }, _context) => {
   const { userId, id, detail, resolved } = body;
 
-  // verify user is allowed to edit work packages
+  // verify user is allowed to edit risk
   const user = await prisma.user.findUnique({ where: { userId } });
   if (!user) return buildNotFoundResponse('user', `#${userId}`);
   if (user.role === Role.GUEST) return buildNoAuthResponse();
 
   // get the original risk and check if it exists
-  const originalRisk = await prisma.risk.findUnique({
-    where: { id }
-  });
-  if (originalRisk === null) {
-    return buildNotFoundResponse('Risk', `#${id}`);
-  }
+  const originalRisk = await prisma.risk.findUnique({ where: { id } });
+  if (!originalRisk) return buildNotFoundResponse('Risk', `#${id}`);
 
   // update the risk with the input fields
   const updatedRisk = await prisma.risk.update({
-    where: {
-      id
-    },
-    data: {
-      detail,
-      isResolved: resolved
-    }
+    where: { id },
+    data: { detail, isResolved: resolved }
   });
 
   // return the updated risk
-  return buildSuccessResponse(updatedRisk);
+  return buildSuccessResponse({ detail: updatedRisk.detail, isResolved: updatedRisk.isResolved });
```

Comment:
I was not 100% sure if this is what James wanted me to do. I referred to #709 ticket and thought that we would only need a field of detail and isResolved for updated risk in the front-end. Correct me if I'm wrong with this transformer.

Reply:
a transformer would be a function, but that's a good start. without further tweaks to the front-end though, I don't think the response would be used for anything

Reply:
yeah the transformer is actually unnecessary here since this isn't a GET route but that's the right idea